src/Promitor.Integrations.AzureMonitor/Promitor.Integrations.AzureMonitor.csproj

```diff
@@ -22,6 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Promitor.Agents.Core\Promitor.Agents.Core.csproj" />
```

Comment:
What is it that you need from agent core here? Normally it should need the reference